ptx/sec_deriv_intro.ptx

```diff
@@ -997,7 +1061,16 @@
           the lines (and hence the tangent lines) abruptly changes at <m>x=0</m>.</caption>
           <!-- START figures/fig_absolutevalue.tex -->
           <image xml:id="img_absolutevalue" width="47%">
-            <description></description>
+            <shortdescription>
+              The graph of function of absolute value of <m>x</m>, the function forms a v shape with it's vertex at origin.
```

Comment:
math tags... I will stop pointing these out, but make sure to check all of the short descriptions.